StructuralPreprocessing/StructuralPreprocessing.XNAT_MARK_RUNNING_STATUS

```diff
@@ -49,10 +49,14 @@ get_options()
 	local arguments=($@)
 
 	# initialize global output variables
-	unset g_project
-	unset g_subject
-	unset g_classifier
-	unset g_running
+    unset g_user
```

Comment:
Same comment as for the FunctionalPreprocessing version of this file. Convert to tabs for indentation so we can be consistent.

Reply:
converted to tabs